matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/database/query/EventEntityQueries.kt

```diff
@@ -42,12 +43,36 @@ internal fun EventEntity.Companion.where(realm: Realm,
         query.equalTo(EventEntityFields.TYPE, type)
     }
     return when (linkFilterMode) {
-        LINKED_ONLY   -> query.equalTo(EventEntityFields.IS_UNLINKED, false)
+        LINKED_ONLY -> query.equalTo(EventEntityFields.IS_UNLINKED, false)
```

Comment:
About formatting please configure AndroidStudio to align the `->`

Reply:
updated my code style preferences

Reply:
thx